src/Resources/skeleton/authenticator/Security52LoginFormAuthenticator.tpl.php

```diff
@@ -22,7 +22,7 @@ class <?= $class_name; ?> extends AbstractLoginFormAuthenticator
 
     public const LOGIN_ROUTE = 'app_login';
 
-    private <?= $use_typed_properties ? 'UrlGeneratorInterface ' : null ?>$urlGenerator;
+    private <?= $is_typed ? 'UrlGeneratorInterface ' : null ?>$urlGenerator;
```

Comment:
because `$use_typed_properties` is a global var that is available in all templates, we should use that to check if properties can be typed.